### PR TITLE
Fix Empty Field Filter

### PR DESCRIPTION
**Pull request by gashach**
Empty Field Filter needs a value when creating pipeline cursor.

### CODE REVIEW

File: src/EncompassRest/Filters/EmptyFieldFilter.cs [hunk 1]
```diff
@@ -4,6 +4,8 @@ namespace EncompassRest.Filters
 {
     public sealed class EmptyFieldFilter : FieldFilter
     {
+        public object Value => System.DateTime.MinValue;
```
Comment (TylerBrinkley) [inline src/EncompassRest/Filters/EmptyFieldFilter.cs]: Make it private and add a `JsonPropertyAttribute` to it.

Comment (TylerBrinkley) [reply src/EncompassRest/Filters/EmptyFieldFilter.cs]: Also add it for `NotEmptyFieldFilter`

**Comment by TylerBrinkley** [comment]
Do the unit tests pass?

**Comment by hfcjweinstock** [comment]
all unit tests pass

**Pull request merged**